native/Avalonia.Native/src/OSX/AvnString.mm

```diff
@@ -153,3 +153,20 @@ virtual HRESULT Get(unsigned int index, IAvnString**ppv) override
 {
     return new AvnStringImpl(data, len);
 }
+
+NSString* GetNSStringAndRelease(IAvnString* s)
+{
+    if (s != nullptr)
+    {
+        char* p;
+        
+        if (s->Pointer((void**)&p) == S_OK && p != nullptr)
+        {
+            return [NSString stringWithUTF8String:p];
+        }
+        
+        s->Release();
```

Comment:
Wait... This never releases when the `NSString` returns above?

Reply:
Yeah, looks like a mistake 👍